src/api/db/migrate/20161128115942_add_when_attribute_to_bs_request.rb

```diff
@@ -4,18 +4,13 @@ class TempBsRequest < ActiveRecord::Base
   end
 
   def self.up
-    add_column :bs_requests, :updated_when, :datetime
-    TempBsRequest.find_each do |bs_request|
-      bs_request.updated_when = bs_request.updated_at
-      bs_request.save!
+    ActiveRecord::Base.transaction do
```

Comment:
You won't be able to rerun the migration as the column is already created and it will fail there.